.npmrc

```diff
@@ -1 +1,2 @@
-save-exact = true
\ No newline at end of file
+save-exact = true
+registry = http://localhost:4873/
```

Comment:
will this not cause a conflict with our release script on github actions?

Reply:
removed

Reply:
thanks!